web/docs/tutorial/07-auth.md

```diff
@@ -31,31 +33,32 @@ psl=}
 
 ## Adding Auth to the Project
 
-Next, we want to tell Wasp that we want to use full-stack [authentication](../auth/overview) in our app:
+Next, tell Wasp to use full-stack [authentication](../auth/overview):
 
-```wasp {7-16} title="main.wasp"
+```wasp title="main.wasp"
 app TodoApp {
   wasp: {
     version: "^0.11.0"
```

Comment:
@sodic Hey what about these versions? We should probably update them to 0.12

Reply:
Yes, we should :smile: 

Don't know how I missed this one.